### PR TITLE
Add first problem

### PR DESCRIPTION
**Pull request by nk-for-insight**

### CODE REVIEW

File: practice_problems_test.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+"""Tests for practice problems."""
+
+import unittest
+
+import practice_problems
+
+class CountAppearancesTest(unittest.TestCase):
+    def test(self):
```
Comment (another-nk-for-insight) [inline practice_problems_test.py]: Please give more descriptive test name, which would contain expected outcome.

Comment (another-nk-for-insight) [reply practice_problems_test.py]: assertDictEqual?

File: practice_problems_test.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+        self.assertEqual(practice_problems.count_appearances([1, 1, 3]), {1: 2, 3: 1})
```
Comment (another-nk-for-insight) [inline practice_problems_test.py]: assertDictEqual?

**Review by another-nk-for-insight — CHANGES_REQUESTED** [review]
Please add bad input handling and appropriate tests

File: practice_problems_test.py (hunk 1 above)
Comment (nk-for-insight) [inline practice_problems_test.py]: could you give an example?

Comment (another-nk-for-insight) [reply practice_problems_test.py]: sure! For example, this test could be test_good_input_results_in_expected_counts

File: practice_problems_test.py (hunk 2 above)
Comment (nk-for-insight) [reply practice_problems_test.py]: Done.

File: practice_problems_test.py (hunk 1 above)
Comment (nk-for-insight) [reply practice_problems_test.py]: Done.

Comment (nk-for-insight) [reply practice_problems_test.py]: Done.

**Comment by nk-for-insight** [comment]
@another-nk-for-insight I think there is no need for bad input handling

**Review by another-nk-for-insight — APPROVED** [review]

**Pull request merged**